python/ql/test/experimental/query-tests/Security-new-dataflow/CWE-502/unsafe_deserialization.py

```diff
@@ -3,14 +3,19 @@
 import yaml
 import marshal
 
+from yaml import SafeLoader
+
 from flask import Flask, request
 app = Flask(__name__)
 
+
 @app.route("/")
 def hello():
-    payload = request.args.get('payload')
-    pickle.loads(payload)  # $getData=payload
-    yaml.load(payload)  # $getData=payload
-    marshal.loads(payload)  # $getData=payload
+    payload = request.args.get("payload")
+    pickle.loads(payload)
+    yaml.load(payload)
+    yaml.load(payload, Loader=SafeLoader)
+    marshal.loads(payload)
```

Comment:
```suggestion
    pickle.loads(payload) # NOT OK
    yaml.load(payload) # NOT OK
    yaml.load(payload, Loader=SafeLoader) # OK
    marshal.loads(payload) # NOT OK
```

I don't think I've added these annotations when I have been writing up query tests, but looking at this code example, it does seem like a good idea. In JS they even have a QL query to check that these annotations are correct, so we might be able to do that as well :wink: